lib/default-wcpm-config.json

```diff
@@ -1,6 +1,7 @@
 {
   "traversalDepth": ["20"],
-  "realms": ["/=karaf"],
+  "guestAccess": ["true"],
+  "sessionAccess": ["true"],
   "authenticationTypes": ["/=IDP|GUEST", "/solr=SAML|PKI|BASIC"],
```

Comment:
This should be `"authenticationTypes": ["/=SAML", "/solr=PKI|BASIC"],` instead. IDP was renamed to be SAML. And old SAML is done by default. If you want to mimic DDF then you may want to add `"/admin=SAML|BASIC"`

Reply:
Will do, thanks!